apps/sms/services/js/moz_mobile_message/messaging_serivce.js

```diff
@@ -0,0 +1,176 @@
+/*global BridgeServiceMixin, Settings */
```

Comment:
typo: ```messaging_service``` (in the name of the file). I don't have strong opinion, but I'd rather have separate folder for messaging service.